src/actions/ConfigurationsActions.ts

```diff
@@ -144,19 +144,26 @@ class ConfigurationsActions extends AbstractActions implements IConfigurationsAc
 
         // Iterating through all values in object
         let objectValue = '';
+        let objectValues = [];
```

Comment:
Generally speaking, the API calls should be separated into a different API abstraction.
All Json manipulation should be separated into a different JSON formatter abstraction, and consider to use JSON.stringify/JSON.parse whenever possible to prevent serialization mistakes.